apps/src/lib/kits/maker/Piezo.js

```diff
@@ -57,3 +57,15 @@ Piezo.prototype.stop = function () {
   five.Piezo.prototype.off.call(this);
 };
 Piezo.prototype.off = Piezo.prototype.stop;
+
+/**
+ * We override johnny-five's note() function because there seems to be a bug
+ * where it computes the frequency incorrectly.
+ * If that bug gets fixed, we should be able to remove this patch.
```

Comment:
nit: might be nice to mention here that we'll also want to remove the tests for the patch